.github/workflows/beam_PerformanceTests_SingleStoreIO.yml

```diff
@@ -15,14 +15,15 @@
 # specific language governing permissions and limitations
 # under the License.
 
-name: Performance Test Java SingleStoreIO 
+name: PerformanceTests SingleStoreIO
```

Comment:
Thanks for unifying the naming. It shows there are a few test still use "Performance Test":

![image](https://github.com/apache/beam/assets/8010435/4f815b9b-8718-488b-ba18-2133bb88100f)

shall we also change them?

Reply:
Similar for a few Load tests:

![image](https://github.com/apache/beam/assets/8010435/f8aae8e1-c691-4070-96b5-4a34024f8ddc)